stl/inc/memory

```diff
@@ -513,6 +588,9 @@ _CONSTEXPR20 void destroy(const _NoThrowFwdIt _First, const _NoThrowFwdIt _Last)
     _Destroy_range(_Get_unwrapped(_First), _Get_unwrapped(_Last));
 }
 
+_EXPORT_STD template <class _ExPo, class _NoThrowFwdIt, _Enable_if_execution_policy_t<_ExPo> = 0>
+void destroy(_ExPo&& _Exec, const _NoThrowFwdIt _First, const _NoThrowFwdIt _Last) noexcept; // terminates

```

Comment:
are we allowed to forward declare like this? If a user simply includes only memory without execution and isn't able to use these algorithms that's a problem. If the can then these are pointless

Reply:
I mirrored forward declarations of parallel algorithms in `<algorithm>`. I guess these forward declarations are conventional.

Reply:
Conventional. The user cannot call parallel algorithms without an execution policy, and the execution policies are defined in `<execution>`. These forward declarations probably aren't strictly necessary, but they are convenient reminders for us maintainers that other overloads of these names exist elsewhere.